style.css

```diff
@@ -0,0 +1,159 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100&display=swap");
+@import url(https://fonts.gstatic.com/s/poppins/v20/pxiEyp8kv8JHgFVrJJbecmNE.woff2);
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
+}
+
+/* navbar styles */
+.navbar {
+  width: 100%;
+  height: auto;
+  padding: 10px 12px;
+  left: 0%;
+  right: 0%;
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-around;
```

Comment:
- [ ] I recommend using `justify-content: space-between` to correct the alignment as requested by the previous code reviewer to match the Figma template. 😉 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/221375684-d7b28bb2-3365-46c0-8f44-ab00f6f707b1.png) | ![image](https://user-images.githubusercontent.com/58642949/221375695-391f9936-ec1b-4f79-91e7-f2e510a2eb5b.png) |